src/lib/errors/AlreadyExstError.ts

```diff
@@ -1,5 +1,5 @@
 class AlreadyExstError extends Error {
-  constructor(modelName) {
+  constructor(modelName: String) {
```

Comment:
대문자 S로 시작하는 String으로 정의 하면 에러 안 나나요?
제가 과거에 에러가 났었던 것으로 기억이 나서 남깁니다.

에러가 안 났었더라고 일반적으로 string으로 정의를 합니다.(엄밀히 말하면 두 개는 다른 타입입니다.)
```suggestion
  constructor(modelName: String) {
```

Reply:
저도 string으로 통일하고 있었는데 자동완성을 사용하다가 잘못 적용된 것 같습니다